pkg/cluster/kubernetes/upgrade.go

```diff
@@ -5,6 +5,9 @@
 package kubernetes
 
 import (
+	"fmt"
+
+	"github.com/coreos/go-semver/semver"
```

Comment:
We already had https://github.com/coreos/go-semver, https://github.com/blang/semver, and https://github.com/hashicorp/go-version in our deps. I picked the first one as it seems to least likely to go away.